pcweb/pages/docs/advanced_guide/wrapping_react.py

```diff
@@ -153,6 +140,14 @@ def wrapping_react():
             rx.code("tag"),
             ", which is the name of the React component. ",
         ),
+        doctext(
+            "Reflex can detect if the package you mentionned in ",
+            rx.code("library"),
+            "need to be installed, no config needed. ",
```

Comment:
Maybe this sentence can be reworded to "Reflex will automatically install the `library` you specify if needed."